server.js

```diff
@@ -1,10 +1,18 @@
 const express = require("express");
-const cors = require('cors')
+const cors = require('cors');
+const path = require('path');
 const app = express();
 const port = process.env.PORT || 3000;
 
+const static = path.resolve(__dirname, 'dist');
+const index = path.resolve(static, 'index.html');
+
 app.use(cors());
-app.use(express.static("dist"));
+app.use(express.static(static));
+
+app.get('*', (req, res) => {
```

Comment:
That may be a bit too generic. What I'm worried about that some static files that genuinely are missing may be falsely reported as HTTP 200 with HTML returned instead - which would make stuff harder to debug.

Reply:
Good point! Do you have any ideas on another way to handle this?

Reply:
Thanks for the help!

Reply:
After some research I think I liked this approach the best:

```js
const handler = (req, res) => res.send(path.join(__dirname, "path/to/your/index.html"))

const routes = ["/", "/hello", "/world"]

routes.forEach( route => app.get(route, handler) )
```

Reply:
After trying this setup, while it is more specific it ends up being a little brittle in our setup because of the need to specify routes. Im going to get this fix out as it is right now to solve the issue and then look into a better fix. Thanks for all the help!